collector/database/classes/collector.php

```diff
@@ -143,7 +143,7 @@ public function get_metrics_aggregated($metricnames = null, $since = false, int
         } else {
             $metricselect = '';
             foreach ($params as $param) {
-                $metricselect .= "AVG(CASE name WHEN '$param' THEN" . $DB->sql_cast_char2int('value', true) . "END) $param,";
+                $metricselect .= "AVG(CASE name WHEN '$param' THEN" . $DB->sql_cast_char2int('value', true) . "END) as \"$param\",";
```

Comment:
[Moodle SQL coding style](https://moodledev.io/general/development/policies/codingstyle/sql) specifies that all SQL keywords should be upper-case. Can you update it to "AS" please :)

Reply:
Thanks for pointing this out. I have updated these keywords to be uppercase.